bin/build.sh

```diff
@@ -12,7 +12,7 @@ if [ "$0" == "./bin/build.sh" -a -f index.ts ]; then
         echo "Please install deno before attempting to build."
         echo "Run: curl -fsSL https://deno.land/install.sh | sh"
     else
-        DENO_DIR=./deno deno compile -o taq --allow-run --allow-write --allow-read --allow-env --import-map ./import_map.json --no-prompt index.ts --quickstart "`cat quickstart.md`" --setBuild "$BUILD" --setVersion "$TAQ_VERSION"
+        DENO_DIR=./deno deno compile -o taq --allow-run --allow-write --allow-read --allow-env --allow-net --import-map ./import_map.json --no-prompt index.ts --quickstart "`cat quickstart.md`" --setBuild "$BUILD" --setVersion "$TAQ_VERSION"
```

Comment:
Jimmy, do you mind also updating the .github/workflows/main.yml please to include the --allow-net flag, as well as the bin/debug.sh script please?